### PR TITLE
chore(#1150): flag-gate multi-site at validate/add; bundle hard-fail points at #1169

### LINKED ISSUE #1150: fix: multi-site is half-supported — dev runs, vibew bundle hard-fails. Finish or flag-gate.
## Problem

From the retro:

> Half-supported multi-site. `AGENTS-VIBEWARDEN` says multi-site works in dev but `vibew bundle` hard-fails on it ("deferred"). That's a feature that exists enough to write code against, then fails when you try to ship. Either finish it or hide it behind a flag until parity. Half-features generate the worst kind of bug report.

The current `vibew bundle` aborts with `multi-site bundle is not yet supported (see ADR-...)`, but the dev path lets users build a multi-site config in the first place. An agent following AGENTS-VIBEWARDEN can scaffold a multi-site setup and then hit a wall at deploy time.

## Proposal

Pick one direction:

**Option A — finish the work**: implement multi-site bundle support. This is real work — the bundle generator needs to produce one compose file with multiple Caddy site stanzas, generate per-site image refs, and the deploy README needs to handle the multi-site case.

**Option B — flag-gate at the dev stage**: refuse multi-site config in `vibew add` / `vibew init` / config validation until bundle support lands. Print a clear "multi-site is post-stable; track at #ADR-XXX". This stops users from getting stuck.

## Recommendation

Option B for now (it's a label/docs change). Option A as a tracked epic post-stable.

## Acceptance criteria (Option B path)

- `vibew validate` (per G / #1144) refuses a multi-site config with a clear rationale and a tracking link.
- `AGENTS-VIBEWARDEN.md` removes any mention of multi-site as a current feature; references it as post-stable only.
- `vibew bundle` keeps its current hard-fail message but its rationale is now consistent ("multi-site is not yet supported; bundles will accept it once X").

## Anchored by (policy reference)

- CLAUDE.md §Architecture principles → Artifact policy (originally seeded as #1137; closed via PR #1152).

## Source

- `~/notes/vibewarden/retro-0.17.0.md`, agent additional drop #7.

### PR DESCRIPTION
**Pull request by tibtof**
Closes #1150

## Summary

- New package `internal/app/multisite/` with `IsProject(configPath string) bool` — single shared detector for all three command surfaces.
- `vibew validate` emits `FAIL  multi-site bundle is post-v1 (see #1169 ...)` on stderr and exits 1 when the project root contains `sites/<name>/vibewarden.yaml`.
- `vibew add` (all subcommands) refuses via `PersistentPreRunE` on the `add` group with message referencing #1169.
- `vibew bundle` `multiSiteErrorMessage` constant updated from the old ADR-085 reference to `#1169`.
- `internal/cli/cmd/bundle.go` local `isMultiSiteProject` function removed; replaced with `multisiteapp.IsProject` call.
- `internal/cli/templates/agents/agents-vibewarden.md.tmpl` and `docs/examples/AGENTS-VIBEWARDEN.md` both receive the architect's verbatim "Multi-site (post-v1)" section; old "deferred; hard-fails today" text and "may have edge cases" Known Limitations bullet removed.
- `CHANGELOG.md` `[Unreleased] ### Changed` entry added.

## Test plan

- `internal/app/multisite/multisite_test.go` — 6-case table covering: no sites dir, empty sites dir, subdir with no YAML, populated subdir, mixed (empty + populated), file at sites/ level.
- `TestValidateCmd_MultiSite_FailRow` — multi-site fixture → exit 1, stderr contains `FAIL`, `post-v1`, `#1169`.
- `TestAddCmd_MultiSite_Refused` — `vibew add tls --domain x.example.com` on multi-site fixture → non-zero exit, output contains `post-v1` and `#1169`.
- `TestBundleCmd_MultiSite_HardErrors` updated — asserts `#1169` instead of old `ADR-085`.
- `make check` passes (gofmt + golangci-lint + build + race tests).

### CODE REVIEW

File: internal/cli/cmd/validate.go [hunk 1]
```diff
@@ -122,6 +123,19 @@ Examples:
...
 			}
 
+			// Multi-site bundle is post-v1 (#1169). Emit a FAIL row consistent
+			// with the OK/OFF/FAIL convention from #1143/#1159 and exit 1 so
+			// that agents and scripts detect the limitation immediately.
+			checkPath := configPath
+			if checkPath == "" {
+				checkPath = "vibewarden.yaml"
+			}
+			absCheck, _ := filepath.Abs(checkPath)
```
Comment (tibtof) [inline internal/cli/cmd/validate.go]: **Swallowed error**: `filepath.Abs` can only fail on Windows when the working directory cannot be retrieved, but the error is still silently discarded (`absCheck, _ := filepath.Abs(checkPath)`). CLAUDE.md mandates 'No swallowed errors'. Change to:

```go
absCheck, err := filepath.Abs(checkPath)
if err != nil {
    absCheck = checkPath
}
```

The fallback is the same behaviour, but the intent is explicit and the rule is satisfied.

**Comment by tibtof** [comment]
**Reviewer Agent: CHANGES REQUESTED**

One violation of CLAUDE.md 'No swallowed errors' rule:

- `internal/cli/cmd/validate.go` line 133: `absCheck, _ := filepath.Abs(checkPath)` discards the error. Fix: capture the error and fall back to `checkPath` explicitly (inline comment posted).

Everything else passes:
- `internal/app/multisite/` is stdlib-only, correctly placed, not in `internal/cli/cmd/`.
- All three consumers (validate, add, bundle) import the shared detector.
- All three messages contain the literal `#1169`.
- Table-driven unit tests cover the full detection matrix (5 cases).
- validate, add, and bundle integration tests all assert `post-v1` and `#1169`.
- `vibew dev` is untouched — no regression path.
- AGENTS template and static copy both have the new 'Multi-site (post-v1)' section; they agree.
- No new ports introduced. No adapter imports in the domain-side helper.
- CHANGELOG entry present.

**Comment by tibtof** [comment]
**Reviewer Agent: APPROVED**

Re-review: `filepath.Abs` error in `internal/cli/cmd/validate.go` is now handled and wrapped correctly. All previous blockers resolved.
